doc/user/content/sql/create-connection.md

```diff
@@ -8,181 +8,53 @@ menu:
 
 ---
 
-A connection describes how to connect and authenticate to an external system you want Materialize to read data from. Once created, a connection is **reusable** across multiple [`CREATE SOURCE`](/sql/create-source) and [`CREATE SINK`](/sql/create-sink) connections.
+A connection describes how to connect and authenticate to an external system you
+want Materialize to read data from. Once created, a connection is **reusable**
+across multiple [`CREATE SOURCE`](/sql/create-source) and [`CREATE SINK`](/sql/create-sink)
+statements.
 
-To use credentials that contain sensitive information (like passwords and SSL keys) in a connection, you must first [create secrets](/sql/create-secret) to securely store each credential in Materialize's secret management system. Credentials that are generally not sensitive (like usernames and SSL certificates) can be specified as plain `text`, or also stored as secrets.
+To use credentials that contain sensitive information (like passwords and SSL
+keys) in a connection, you must first [create secrets](/sql/create-secret) to
+securely store each credential in Materialize's secret management system.
+Credentials that are generally not sensitive (like usernames and SSL
+certificates) can be specified as plain `text`, or also stored as secrets.
 
-## Syntax
+## Connectors
```

Comment:
The outcome of the grouping makes sense, but I'm not sold on the name "connectors"! Talking about the "AWS PrivateLink connector" or the "SSH tunnel connector" type checks for me. In fact I could have sworn you were the one to tell me that a "connector" is a shorter way of saying "connection type."

Strawman, but the best alternative I can come up with is "source and sink connectors" vs "network connectors."

Reply:
We use "connector" in the [source](https://materialize.com/docs/sql/create-source/) and [sink](https://materialize.com/docs/sql/create-sink/) documentation to refer to each external system-specific implementation in a neutral way, but this predates PrivateLink and SSH connections. The Confluent Schema Registry connection type also sits in there like a bit of an odd duckling. 

Sigh, I'll sit on it a bit.